pipeline-examples/configfile-provider-plugin/README.md

```diff
@@ -0,0 +1 @@
+An example showing howto use configFile Provider plugin.
```

Comment:
Would you mind adding more content here, such as a link to the configFile Provider plugin on plugins.jenkins.io, and perhaps a nice succinct description of what the use-case would be for configFile Provider in Pipeline?

The READMEs for these examples are automatically collected and published [to jenkins.io](https://jenkins.io/doc/pipeline/examples/) so more information is better.